app/api/__init__.py

```diff
@@ -20,14 +20,13 @@
 from app.api.settings import SettingDetail
 from app.api.discount_codes import DiscountCodeList, DiscountCodeDetail, DiscountCodeRelationship
 
-
 api_v1 = Blueprint('v1', __name__, url_prefix='/v1')
 api = Api(app, api_v1)
 
 # users
 api.route(UserList, 'user_list', '/users')
-api.route(UserDetail, 'user_detail', '/users/<int:id>',
-          '/notifications/<int:notification_id>/user', '/event-invoices/<int:event_invoice_id>/user')
+api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user',
```

Comment:
Should be `events/<id or identifier>/users` ... Plural. One event would be related to many users.